sessions/backend-update-delete/demo-end/components/JokeForm/index.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import Button from "../Button";
+import { useState } from "react";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin-block-start: 2rem;
```

Comment:
nice! (just learned about this in the josh comeau Kurs)

Reply:
Modern CSS (@container, @layer, :has, :is, etc ...) is extremely fun and useful - I love it sm.

Reply:
maybe you can give a short workshop about that? I have no idea what you are talking about :D

Reply:
Nah 😂